packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -336,9 +334,10 @@ class ScaffoldState extends State<Scaffold> {
       ));
     }
 
-    return new CustomMultiChildLayout(children, delegate: _scaffoldLayout);
+    return new CustomMultiChildLayout(children, delegate: new _ScaffoldLayout());
   }
 
+  bool shouldRelayout(MultiChildLayoutDelegate oldDelegate) => true;
```

Comment:
though since you inherit one that just returns true, you can probably just not have it. I'm assuming here that we do want this guy to always relayout regardless.